src/bp/diag/index.ts

```diff
@@ -68,7 +70,10 @@ export const ENV_VARS = [
 ]
 
 const PASSWORD_REGEX = new RegExp(/(.*):(.*)@(.*)/)
+const REDIS_TEST_KEY = 'botpress_redis_test_key'
+const REDIS_TEST_VALUE = 'bp123'
```

Comment:
perhaps make it random so that a previous set (unlikely as the delete _also_ didn't work) doesn't trick into thinking it worked now